about.html

```diff
@@ -0,0 +1,100 @@
+            </ul>
+            </div>
+        </div>
+            <div class="desktop-navbar">
+                <a class="logo" href="index.html">
+                    <img id="logo-img" src="./images/logo.png" alt="logo-image">
+                </a>
+            <menu class="desktop-nav-list">
+                <ul>
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="#services-id">Our Services</a></li>
+                    <li><a href="#team-id">Team Members</a></li>
+                    <li><a href="#team-id">Sponser</a></li> 
+                </ul>
+            </menu>
+        </div>
+            </nav>
+        <div class="about-detail">
+            <div class="about-heading">
+                <h1>An Engineering, Procurement & Construction Company</h1>
+                <h2 id="company">About Our Company</h2>
+            </div>
+            <div class="about-description">
+                <p id="about-dec">R&R is committed to excel in the industry with responsibility and pride. With an industry-leading expert team, R&R is in a position to execute infrastructure projects more efficiently. R&R is looking to expand its portfolio in power systems, transportation and civil infrastructure. R&R is a well-established company and has a very strong technical profile of engineering, procurement, and construction of power grid stations and transmission lines.</p>
+            </div>
+            <div class="contact">
+                <p>We are always looking for new relationships whilst strengthening the existing relationships.</p><br>
+                <p>Please reach us out through <a href="mailto:someone@example.com">email</a></p>
+            </div>
+        </div>
+        </section>
+        <section class="previous-projects">
+            <div class="projects-heading">
+                <p>Our recent completed Projects</p>
+                <p id="about-project">Take a look at our recently completed projects in Dubai and Doha.</p>
+            </div>
+            <div class="projects">
+                <div class="project1">
+                    <img class="project-img" src="./images/project1.jfif" alt="Project1-image">
+                    <h4 class="project-detail">Doha Transmission Line Project Completed in 2017</h4>
+                </div>
+                <div class="project2">
+                    <img class="project-img" src="./images/project2.jpg" alt="project2-image">
+                    <h4 class="project-detail">Dubai Grid station Project Completed in 2021</h4>
+                </div>
+            </div>
+        </section>
+        <footer class="footer-about">
+            <div class="footer-logo-about">
+                <img src="./images/logo.png" alt="logo-image">
+            </div>
+            <div class="footer-description-about">
+                <p>2022 R&R Some Rights Reserved.</p>
+            </div>
+        </footer>
+    <script src="script.js"></script>   
```

Comment:
Kindly check and fix some of the discrepancies shown below in the about page. Check the margins, paddings, width etc. All page elements should have similar spacing, borders, shadows, and paddings.

Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-10-22 17-56-15](https://user-images.githubusercontent.com/65233860/197355877-d10f8628-5b9e-4f09-a04a-b07e3f88b65d.png)  |  ![Screenshot from 2022-10-22 17-49-34](https://user-images.githubusercontent.com/65233860/197355883-702fd054-0783-438e-9cfb-9ea43265d646.png)


Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-10-22 17-36-37](https://user-images.githubusercontent.com/65233860/197355305-1e794ae3-6e66-4acd-9d87-0764195f0c28.png)  |  ![Screenshot from 2022-10-22 17-49-34](https://user-images.githubusercontent.com/65233860/197355345-4e85c44b-fcde-4a66-840f-ffac86736471.png)